.github/workflows/cr.yml

```diff
@@ -7,11 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: corepack enable
+      - uses: pnpm/action-setup@v4
       - uses: actions/setup-node@v4
         with:
           node-version: 20
           cache: 'pnpm'
-      - run: pnpm install
+          cache-dependency-path: '**/pnpm-lock.yaml'
+      - run: pnpm install --frozen-lockfile
```

Comment:
frozen-lockfile is the default in CI, so let's remove it. https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data